### PR TITLE
Add Emoji Log

### PR DESCRIPTION
**Pull request by rosano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@
 - [HabitBull](http://www.habitbull.com/) - Premium habit tracker.
 - [Loop Habit Tracker](https://github.com/iSoron/uhabits) - An Android app for creating and maintaining good habits.
 - [Waka Time](https://wakatime.com/) -  Open source plugin for knowing exactly how long you spend coding.
+- [Emoji Log](https://emojilog.rosano.ca) -  Non-streak emoji-based tracker for arbitrary things.
```
Comment (jyguyomarch) [inline README.md:93]: Move your entry up to keep the alphabetical order. Also not sure what you mean by "non-streak" here. Maybe room to make the description a bit clearer?

**Review by jyguyomarch — CHANGES_REQUESTED** [review]

**Comment by jyguyomarch** [comment]
Pretty cool app @rosano 👍

File: README.md (hunk 1 above)
Comment (rosano) [reply README.md]: Done. I was following https://github.com/jyguyomarch/awesome-productivity/blob/82853769a34c435736e9c0fdf84199e8e44e4e6f/CONTRIBUTING.md?plain=1#L10

**Comment by rosano** [comment]
@jyguyomarch thank you! If you consider https://hyperdraft.rosano.ca relevant I can add it as well: it combines markdown note-taking with website building + you own your data 100%.

**Review by jyguyomarch — APPROVED** [review]

**Comment by jyguyomarch** [comment]
@rosano Hyperdraft feels a bit light on the Note taking or Knowledge Management side for this list compared to other solution. But the self hosting and the possibility to easily publish makes me think it could be a good fit for other awesome list. Maybe:
- https://github.com/awesome-selfhosted/awesome-selfhosted
- https://github.com/ripienaar/free-for-dev
- https://github.com/TalAter/awesome-book-authoring
- https://github.com/emptymalei/awesome-research

**Pull request merged**